client/src/app/core/services/choice.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { OpenSlidesComponent } from '../../openslides.component';
+import { MatDialog } from '@angular/material';
+import { ChoiceDialogComponent } from '../../shared/components/choice-dialog/choice-dialog.component';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ChoiceService extends OpenSlidesComponent {
+
+    public constructor(private dialog: MatDialog) {
+        super();
+    }
+    /**
+     * Opens the dialog. Returns true, if the user accepts.
+     * @param title The title to display in the dialog
+     * @param choices The available choices (object with id and label)
+     */
+    public async open(title: string, choices: Object): Promise<any> {
```

Comment:
missing @returns, if possible, avoid the return of `any`
we should specify what the answer of the dialog could be.

I noticed that a lot of internal components are using the async await promise feature.